### ISSUE TITLE
Caret is a rectangle with Windows High Contrast Mode

### ISSUE THREAD

**Issue by Zauberbutter** [twbs/bootstrap#27650]
# Information

- Operating system and version: Windows 10 Home
- Browser and version: Firefox 63.0.1 & IE 11.1.17134.0

# Bug

When using transparent borders they aren't transparent with Windows High Contrast Mode, so you can't see that `caret` is a caret:

![drodpown](https://user-images.githubusercontent.com/16780924/48312185-3f5c9300-e5ab-11e8-88d0-18b03f77610c.png)

# Steps to reproduce

1. Open Firefox or IE
2. Navigate to http://getbootstrap.com/docs/4.0/components/dropdowns/
3. Enable Windows High Contrast Mode `(left) shift + (left) alt + print`

**Comment by patrickhlauke** [comment]
more generally, we should probably consider using SVG over any border-based things for WHCM

**Comment by mdo** [comment]
Confirming (hella late) that this is still an issue in v4.5.x and v5.x.

**Comment by Cerberooo** [comment]
@mdo Do you have a solution in mind? Maybe using SVG in `content` of a pseudo class?

**Comment by mdo** [comment]
One solution I've cooked up so far is changing how our dropdown carets look: https://codepen.io/emdeoh/pen/gOwQPXq. This new one shows up nicely for me, but would need some more testing.

**Comment by ffoodd** [comment]
Simple note here, I'm used to handle carets with `clip-path` instead of borders or SVG.

@mdo Your suggestion is pretty clever, however won't it be confusingly similar to standard `select` in some browsers and systems?

**Comment by mdo** [comment]
Not as concerned with that, but it's an interesting concern. I think having a consistent visual indicator of "more" for any interactive element is okay?

**Comment by mdo** [comment]
Unclear if `clip-path` has the best browser support... definitely doesn't work in IE11 for sure.

Updated https://cdpn.io/emdeoh/pen/gOwQPXq with a `clip-path` example regardless.

**Comment by patrickhlauke** [comment]
personally still feel SVG would give more flexibility, but yeah ... border/clip path trickery is cute too

**Comment by patrickhlauke** [comment]
x-ref https://github.com/twbs/bootstrap/issues/35941
